components/Nav/Navbar/Popouts/SettingsPopout.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useCallback, useState } from 'react';
+import { useRouter } from 'next/router';
+import shallow from 'zustand/shallow';
+import {
+    Slider,
+    VersionSelector,
+    ToggleSwitch,
+    SwitchOption,
+    DarkModeSelector,
+    StyledSettingsPopout,
+} from '~/components/Nav/Navbar/Popouts/styles';
+import { useStore } from '~/store/main';
+import { selectThemeSlice } from '~/store/ThemeSlice';
+
+const SettingsPopout: React.FC<{ isActive: boolean }> = ({ isActive }) => {
+    const router = useRouter();
+    const [v2selected, setV2Selected] = useState(true);
+    const { isDark, toggleTheme } = useStore(selectThemeSlice, shallow);
+    const basePath = 'https://poolsv1.tracer.finance';
+
+    const getRoute = useCallback(() => {
+        if (router.route === '/') {
+            return `${basePath}/pools`;
+        } else {
+            // assumes that the destination base path appropriately handles redirects
+            return `${basePath}${router.route}`;
+        }
+    }, [router.route]);
+
+    const handleVersionSwitch = () => {
+        setV2Selected(!v2selected);
+        // Wait for toggle animation to complete before navigating to v1
+        setTimeout(() => {
+            open(getRoute(), '_self');
+        }, 600);
```

Comment:
is there any way to get this 600 as a variable sourced from the animation itself? assume if we ever change the animation duration this will no longer lineup

Reply:
Updated with constant variable